projects/CenterPoint/configs/t4dataset/Centerpoint/second_secfpn_4xb16_121m_base_amp_t4metric_v2.py

```diff
@@ -0,0 +1,451 @@
+# type, data_root and ann_file of data.train, data.val and data.test
+point_cloud_range = [-121.60, -121.60, -3.0, 121.60, 121.60, 5.0]
+voxel_size = [0.32, 0.32, 8.0]
+grid_size = [760, 760, 1]  # (121.60 / 0.32 == 380, 380 * 2 == 760)
+sweeps_num = 1
+input_modality = dict(
+    use_lidar=True,
+    use_camera=False,
+    use_radar=False,
+    use_map=False,
+    use_external=False,
+)
+out_size_factor = 1
+
+backend_args = None
+# backend_args = dict(backend="disk")
+point_load_dim = 5  # x, y, z, intensity, ring_id
+point_use_dim = 3  # x, y, z
+lidar_sweep_dims = [0, 1, 2, 4]
+
+# eval parameter
+eval_class_range = {
+    "car": 121,
+    "truck": 121,
+    "bus": 121,
+    "bicycle": 121,
+    "pedestrian": 121,
+}
+
+# user setting
+data_root = "data/t4dataset/"
+info_directory_path = "info/user_name/"
+train_gpu_size = 4
+train_batch_size = 16
+test_batch_size = 2
+num_workers = 32
+val_interval = 5
+max_epochs = 50
+work_dir = "work_dirs/centerpoint/" + _base_.dataset_type + "/second_secfpn_4xb16_121m_base_amp_t4metric_v2/"
+
+train_pipeline = [
+    dict(
+        type="LoadPointsFromFile",
+        coord_type="LIDAR",
+        load_dim=point_load_dim,
+        use_dim=point_load_dim,
+        backend_args=backend_args,
+    ),
+    dict(
+        type="LoadPointsFromMultiSweeps",
+        sweeps_num=sweeps_num,
+        load_dim=point_load_dim,
+        use_dim=lidar_sweep_dims,
+        pad_empty_sweeps=True,
+        remove_close=True,
+        backend_args=backend_args,
+    ),
+    dict(type="LoadAnnotations3D", with_bbox_3d=True, with_label_3d=True),
+    dict(
+        type="RandomFlip3D",
+        sync_2d=False,
+        flip_ratio_bev_horizontal=0.5,
+        flip_ratio_bev_vertical=0.5,
+    ),
+    dict(
+        type="GlobalRotScaleTrans",
+        rot_range=[-1.571, 1.571],
+        scale_ratio_range=[0.80, 1.20],
+        translation_std=[1.0, 1.0, 0.2],
+    ),
+    dict(type="PointsRangeFilter", point_cloud_range=point_cloud_range),
+    dict(type="ObjectRangeFilter", point_cloud_range=point_cloud_range),
+    dict(type="ObjectNameFilter", classes={{_base_.class_names}}),
+    dict(type="ObjectMinPointsFilter", min_num_points=5),
+    dict(type="PointShuffle"),
+    dict(type="Pack3DDetInputs", keys=["points", "gt_bboxes_3d", "gt_labels_3d"]),
+]
+
+test_pipeline = [
+    dict(
+        type="LoadPointsFromFile",
+        coord_type="LIDAR",
+        load_dim=point_load_dim,
+        use_dim=point_load_dim,
+        backend_args=backend_args,
+    ),
+    dict(
+        type="LoadPointsFromMultiSweeps",
+        sweeps_num=sweeps_num,
+        load_dim=point_load_dim,
+        use_dim=lidar_sweep_dims,
+        pad_empty_sweeps=True,
+        remove_close=True,
+        backend_args=backend_args,
+    ),
+    dict(type="PointsRangeFilter", point_cloud_range=point_cloud_range),
+    dict(
+        type="Pack3DDetInputs",
+        keys=["points", "gt_bboxes_3d", "gt_labels_3d"],
+        meta_keys=(
```

Comment:
```suggestion
        # Specify the metadata keys required by the downstream pipeline.
        # Refer to the official MMDetection3D formatting transform implementation for details:
        # https://github.com/open-mmlab/mmdetection3d/blob/main/mmdet3d/datasets/transforms/formating.py#L67
        # Also see the content structure in "t4dataset_base_infos_test.pkl" for reference.
        meta_keys=(
```

Maybe we should keep the comment?